be/src/runtime/exec_env.cpp

```diff
@@ -462,6 +465,10 @@ void ExecEnv::_destroy() {
         delete _query_pool_mem_tracker;
         _query_pool_mem_tracker = nullptr;
     }
+    if (_query_context_mgr) {
+        delete _query_context_mgr;
```

Comment:
Maybe they could be replaced as `unique_ptr`?

Reply:
in ExecEnv,  other objects are raw pointers, keep the code style uniformly.